server/db/models/user.js

```diff
@@ -6,11 +6,11 @@ const User = db.define('user', {
   email: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
-  },
-  password: {
-    type: Sequelize.STRING
+    validate: {
+      isEmail: true
+    }
   },
+  password: Sequelize.STRING,
```

Comment:
Perhaps we could add a custom validator function to check and see if there is a facebookId or googleid, and if not, require a password